### PR TITLE
Support Tools interface

### PR DESCRIPTION
**Pull request by jonahkagan**
Adds a new Support Tools interface (formerly known as the superadmin interface) built in React.

Features:
- Home screen with list of organizations
- Org screen with list of audits, list of audit admins (with login buttons), and form to create new audit admin (including integrating with Auth0)
- Audit screen with list of jurisdictions and list of jurisdiction admins (with login buttons)

I took this opportunity to test out two new libraries to help with data loading (react-query) and forms (react-hook-form).

https://user-images.githubusercontent.com/530106/103838547-31e3ac00-5042-11eb-9fb3-41588a7d9e26.mov

### CODE REVIEW

File: server/api/support.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    token = auth0_get_token()
+    auth0 = Auth0(AUTH0_DOMAIN, token)
+    try:
+        auth0.users.create(
+            dict(
+                email=email,
+                password=secrets.token_urlsafe(),
+                connection="Username-Password-Authentication",
+            )
+        )
```
Comment (benadida) [inline server/api/support.py:47]: should we store an `external_user_id`, or is the user keyed by `email` in the Auth0 DB?

**Review by benadida — APPROVED** [review]
one question about ensuring that we have a proper reference to the auth0 user row in our database, otherwise good!

File: server/api/support.py (hunk 1 above)
Comment (jonahkagan) [reply server/api/support.py]: added here: 972b1d4

File: server/api/support.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        return str(user["user_id"])
+    except Auth0Error as error:
+        # If user already exists in Auth0, no problem!
+        if error.status_code == 409:
+            users = auth0.users_by_email.search_users_by_email(email.lower())
+            return str(users[0]["user_id"])
```
Comment (benadida) [inline server/api/support.py:53]: What if the user is not in our DB? Unlikely given the sequence of calls, but probably something to check.

**Review by benadida — APPROVED** [review]
one small error check to add, otherwise good.

File: server/api/support.py (hunk 2 above)
Comment (benadida) [reply server/api/support.py:53]: read the first line wrong. Ignore.

**Pull request merged**